src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    public abstract boolean doWork();

```

Comment:
method should return nothing
```suggestion
    public abstract void doWork();

```